packit/config/job_config.py

```diff
@@ -60,6 +60,7 @@ def __init__(
         project: str = None,
         dist_git_branch: str = None,
         branch: str = None,
+        scratch: bool = None,
```

Comment:
I'm wondering how is `None` different from `False`.
Could it be `scratch: bool = False,` instead?

Reply:
Explicitely differentiate between not-set and set value? (I'll change it to `False`.)